dropwizard-hibernate/src/test/java/io/dropwizard/hibernate/AbstractDAOTest.java

```diff
@@ -84,7 +84,7 @@ public String persist(String entity) throws HibernateException {
 
     private final SessionFactory factory = mock(SessionFactory.class);
     private final Criteria criteria = mock(Criteria.class);
-    private final Query query = mock(Query.class);
+    private final org.hibernate.query.Query query = mock(org.hibernate.query.Query.class);
```

Comment:
I think we could also move to `import org.hibernate.query.Query` in `AsbtractDAO` instead of deprecated `org.hibernate.Query`.